lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1233,3 +1564,42 @@ spec = do
                 "passphrase": "Secure Passphrase"
             }|]
         return (wSrc, payload)
+
+    fixtureWalletManyTxs
+      :: Context t
+      -> [Natural]
+      -> IO (ApiWallet, ApiWallet, [(UTCTime, Natural)])
+    fixtureWalletManyTxs ctx txAmounts = do
+        (wSrc, wDest) <- (,) <$> fixtureWallet ctx <*> emptyWallet ctx
+        addrs <- listAddresses ctx wDest
+
+        forM_ txAmounts $ \amt -> do
+            let destination = (addrs !! 1) ^. #id
+            let payload = Json [json|{
+                    "payments": [{
+                        "address": #{destination},
+                        "amount": {
+                            "quantity": #{amt},
+                            "unit": "lovelace"
+                        }
+                    }],
+                    "passphrase": "cardano-wallet"
+                }|]
+
+            rg <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
+            let balAv = getFromResponse balanceAvailable rg
+            let balTot = getFromResponse balanceTotal rg
+
+            request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
+              >>= expectResponseCode HTTP.status202
+            expectEventually' ctx balanceAvailable (amt + balAv) wDest
+            expectEventually' ctx balanceTotal (amt + balTot) wDest
+
+        expectEventually' ctx balanceAvailable (sum txAmounts) wDest
+        expectEventually' ctx balanceTotal (sum txAmounts) wDest
+        r <- request @([ApiTransaction t]) ctx (listTxEp wDest "?order=ascending")
+            Default Empty
+        let indexes = [0..(length txAmounts - 1)]
+        let txTimes = map (\x -> fromJust $ getFromResponseList x insertedAtTime r) indexes
```

Comment:
Some lines in this PR are a bit longer than our coding standard allows.

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Perhaps have a quick scan over the files changed in this PR?

For the above case, perhaps we can format like this:
```hs
        let txTimes = map
                (\x -> fromJust $ getFromResponseList x insertedAtTime r)
                indexes
```